db/migrate/20161015070842_setup_njdb.rb

```diff
@@ -13,21 +13,21 @@ def change
       p.string :name
```

Comment:
It's an autogenerated file rails via `bin/rails generate migration MyLabel` that tries to keep it unique (hence the filename format of what I believe is `YYYYMMDDHHMMSS_my_label`)